guides/README.md

```diff
@@ -0,0 +1,27 @@
+# Rails Guides Redesign 2024
+
+## About the Project
+
+The Rails Guides Visual Refresh occured in Q1 2024, and was intended to bring the visual style of the guides inline with the rubyonrails.org site. 
+
+## Editing Depedencies
+
+The editing files for the Guides rebuild reside in `stylesrc` and use SCSS to improve developer experience. The code base relies on `include_media` (https://eduardoboucas.github.io/include-media/) to enable inline media-queries adjustments. We've also relied on the standard `normalize.css` (https://necolas.github.io/normalize.css/) to help bring all browsers together.
+
+## Building the Guides in Development
+
+To generate new guides into static files, type `rake guides:generate` from inside the `guides` folder. If you make changes to the HTML or ERB, you'll need to remove the "output" directory before running this command. The master SCSS files (style.scss, highlight.scss) will compile as part of this process. 
+
+## FAQ
+
+### Why are you not using CSS variables?
+
+Per the MDN documentation on CSS custom properties (https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties), they are not supported in media or container queries at this point (Feb 2024). They may in future releases, and we should pivot to that when they are more wholistically supported. SCSS variables, because they are interpolated at build, serve a similar purpose and allow us the flexibilty to support much older browsers.
+
+### Why do we include LTR and RTL?
+
+LTR/RTL (Left to right/right to left) is a layout change based on the nature of the language the site is being displayed in. Arabic and Farsi are two well known "RTL" languages. If the site is automatically translated, then the layout will adjust (mirror horizontally) to be more in line with the text.
```

Comment:
Just tried RTL, and it causes the guide content to get under the nav on the right, and some of the boxes go over:

![Screenshot 2024-03-18 at 17 33 32](https://github.com/rails/rails/assets/26328/5699fd59-caf3-440a-9c69-14393266c5cc)

It also appears to create some horizontal scrolling, creating a lot of space between the content and the menu (home / guides index / contribute).

Are there plans to work on this support separately?

Reply:
Yes. It's an item we deemed not critical for launch. It was intended for follow up later this week. Ideally with an arabic or farsi-speaking developer where we can check any issues that occur in a real RTL language.